nodeList.json

```diff
@@ -1,875 +1,36 @@
-   },
-   {
-      "name": "Westmount-b274",
-      "latitude": 43.68858897327703,
-      "longitude": -79.51539830214313,
-      "cardinalDirectionAntenna": "Omni",
-      "status":"active",
-      "IPV6Address":"fc58:30df:d758:5c92:dba3:5916:b4ef:b274",
-      "floor": 1,
-      "dateAdded": "2017-10-28T16:58:11.735Z"
-   },
-   {
-      "name": "Node2",
-      "latitude": 43.644728204426315,
-      "longitude":-79.38400492913058,
-      "cardinalDirection": "None",
-      "status":"active",
-      "IPV6Address":"fc6e:691e:dfaa:b992:a10a:7b49:5a1a:5e09",
-      "floor": 1,
-      "dateAdded": "2017-10-28T02:34:57.023Z"
-   },
-   {
-      "name": "Stouffville",
-      "latitude": 43.9705197,
-      "longitude":-79.23860079999997,
-      "cardinalDirection": "West",
-      "cardinalDirectionAntenna": "Omni",
-      "floor": 2,
-      "status":"active",
-      "IPV6Address":"fc52:b22c:0b85:f371:24ab:4572:90b1:d926",
-      "dateAdded": "2017-10-30T01:33:24.703Z"
-   },
-   {
-     "name": "Free Geek Toronto",
-     "latitude": 43.641920944151614,
-     "longitude":-79.4252866754631,
-     "cardinalDirection": "Omni",
-     "floor": 1,
-     "status": "active",
-     "IPV6Address":"fcc5:480e:20d6:dd89:12e1:19f8:23b9:f6ab",
-     "dateAdded": "2018-04-19T23:52:59.804Z"
-   },
-   {
-     "name": "bridlewood-2a13",
-     "latitude": 43.7956074,
-     "longitude":-79.32741190000002,
-     "cardinalDirection": "East",
-     "cardinalDirectionAntenna": "Omni",
-     "floor": 3,
-     "status": "active",
-     "IPV6Address":"fc5a:dc78:698d:2929:9b09:46b:d358:2a13",
-     "dateAdded": "2019-02-27T10:00:48.936Z",
-     "contat": "Turco32"
-   },
-   {
-     "name": "Skymark",
-     "latitude": 43.66290636709295,
-     "longitude":-79.59280571999892,
-     "cardinalDirection": "South East",
-     "floor": 5,
-     "status": "proposed",
-     "dateAdded": "2019-01-14T03:22:19.832Z"
-   },
-   {
-     "name": "LibreMesh-59bf",
-     "latitude": 51.960440646753135,
-     "longitude":5.966175190145464,
-     "cardinalDirection": "Omni",
-     "cardinalDirectionAntenna": "Omni",
-     "floor": 3,
-     "status": "active",
-     "contact": "me@finlaydag33k.nl",
-     "IPV6Address": "fccb:3403:7b11:e7f6:5ecc:8ac7:537d:59bf",
-     "dateAdded": "2019-05-16T11:46:05.311Z"
-   }   
-]
+{
+    "SN1R1": {
+        "type": "router",
+        "latitude": 43.672952480910325,
+        "longitude": -79.49166343457031,
+        "IPv4": "100.64.10.1/24"
+    },
+    "SN1A1": {
+        "type": "antenna",
+        "mode": "AP",
+        "ssid": "TOMESH-AC-SN1A1",
+        "IPv4": "100.64.10.2",
+        "router": "SN1R1",
+        "latitude": 43.67298352127934,
+        "longitude": -79.4918780112915,
+        "antennaDirection":195,
+        "antennaDegreas": 120,
+        "antennaDistance": 2500,
+        "antennaHeight": "170",
+        "antennaType": "Sector"
+    },
+    "SN1A2": {
+        "type": "antenna",
+        "mode": "AP",
+        "ssid": "TOMESH-AC-SN1A2",
```

Comment:
Not sure why we decided to capitalize these. Why not `tomesh-ac-sn1a2`. Also as I pointed out before, we should list the protocol as `airmax` rather than `ac` since this is not 802.11ac.

Reply:
I think once https://github.com/tomeshnet/toronto-community-network/issues/21 gets agreaed upon we can update these names.